lwallet/Pos/Wallet/Light/Redirect.hs

```diff
@@ -40,8 +41,8 @@ getBalanceWallet = getBalanceFromUtxo
 
 getBlockHistoryWallet
     :: (MonadReader ctx m, HasLens LWS.WalletState ctx LWS.WalletState, MonadIO m)
-    => [Address] -> m (DList TxHistoryEntry)
-getBlockHistoryWallet addrs = do
+    => Tagged ssc ([Address] -> m (DList TxHistoryEntry))
+getBlockHistoryWallet = Tagged $ \addrs -> do
```

Comment:
No, I was so happy when I've seen that we can get rid of this `Tagged` shit here in master, please remove it =\\